openfl/protocols/director_pb2_grpc.py

```diff
@@ -5,7 +5,7 @@
 from . import director_pb2 as director__pb2
 
 
-class FederationDirectorStub:
+class FederationDirectorStub(object):
```

Comment:
should we continue python2 support? 😁

Reply:
It's an autogenerated code from proto files.